src/Microsoft.DotNet.Build.Tasks/Microsoft.DotNet.Build.Tasks.csproj

```diff
@@ -82,9 +83,81 @@
   </ItemGroup>
   <ItemGroup>
     <None Include="project.json" />
-    <None Include="PackageFiles\**\*.*" />
+    <None Include="PackageFiles\ApiCompat.runtimeconfig.json" />
```

Comment:
This was almost certainly because I opened the project in VS and it automatically did this.